portafly/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "portafly",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@patternfly/react-core": "^3.124.1",
+    "@patternfly/react-table": "^2.25.6",
+    "react": "^16.12.0",
+    "react-async": "^10.0.0",
+    "react-dom": "^16.12.0",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-router-last-location": "^2.0.1",
+    "react-scripts": "3.3.1",
+    "typescript": "~3.7.2"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-router": "^5.1.3",
+    "@types/react-router-dom": "^5.1.3",
+    "date-fns": "^2.9.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

Comment:
What if we add a script using the env vars we need?
`"dev": "PORT=3003 SKIP_PREFLIGHT_CHECK=true react-scripts start"`

Reply:
that'd be polluting the package.json, and not letting anyone to choose those variables to customize their envs

Reply:
That's true